command/operator_raft_list.go

```diff
@@ -57,24 +58,43 @@ func (c *OperatorRaftListCommand) Run(args []string) int {
 	return 0
 }
 
-func raftListPeers(operator *api.Operator, stale bool) (string, error) {
+func raftListPeers(client *api.Client, stale bool) (string, error) {
+	raftProtocols := make(map[string]string)
+	members, err := client.Agent().Members(false)
+	if err != nil {
+		return "", err
+	}
+
+	for _, member := range members {
+		if serf.MemberStatus(member.Status) == serf.StatusLeft {
+			continue
+		}
+
+		if member.Tags["role"] != "consul" {
+			continue
+		}
+
+		raftProtocols[member.Name] = member.Tags["raft_vsn"]
+	}
+
 	q := &api.QueryOptions{
 		AllowStale: stale,
 	}
-	reply, err := operator.RaftGetConfiguration(q)
+	reply, err := client.Operator().RaftGetConfiguration(q)
 	if err != nil {
 		return "", fmt.Errorf("Failed to retrieve raft configuration: %v", err)
 	}
 
 	// Format it as a nice table.
-	result := []string{"Node|ID|Address|State|Voter"}
+	result := []string{"Node|ID|Address|State|Voter|RaftProtocol"}
 	for _, s := range reply.Servers {
+		raftProtocol := raftProtocols[s.Node]
```

Comment:
How about an "unknown" if the version isn't there. Or we could show "1" even, since that's what older clients are running (or "<=1").